<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientApplication</class>
 <widget class="QMainWindow" name="ClientApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Application</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(157, 187, 190, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>531</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="selectedFilter">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.9, y1:1, x2:1, y2:0, stop:0 rgba(82, 150, 69, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="filterList_2">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.199005 rgba(194, 212, 197, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>480</y>
      <width>271</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextEdit" name="selectFilter">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.199005 rgba(194, 212, 197, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="filterList">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(7, 55, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(255, 255, 255);
font: 12pt &quot;Fixedsys&quot;;
font: 8pt &quot;Goudy Stout&quot;;</string>
    </property>
    <property name="text">
     <string>Filter List</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectedList">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(7, 55, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(255, 255, 255);
font: 12pt &quot;Fixedsys&quot;;
font: 8pt &quot;Goudy Stout&quot;;</string>
    </property>
    <property name="text">
     <string>    Selected List</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>261</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTextEdit" name="deleteFilter">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.199005 rgba(194, 212, 197, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(198, 198, 164);
                font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>SELECT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>450</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(198, 198, 164);
                font: 75 10pt &quot;MS Shell Dlg 2&quot;;
             
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>DELETE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>600</y>
      <width>271</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="selectFilterBtn">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(39, 55, 142, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(198, 198, 164);</string>
       </property>
       <property name="text">
        <string>SELECT FILTER</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>600</y>
      <width>261</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(39, 55, 142, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(198, 198, 164);</string>
       </property>
       <property name="text">
        <string>DELETE FILTER</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QCheckBox" name="filterBox">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Niagara Solid&quot;;
font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(157, 187, 190, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(198, 198, 164);</string>
       </property>
       <property name="text">
        <string>GROUP FILTER</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>640</y>
      <width>531</width>
      <height>59</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="start">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(198, 198, 164);</string>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(39, 55, 142, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(198, 198, 164);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>RESTART</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>700</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QLabel" name="consoleLayer">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionEXIT">
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication4.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>start</sender>
   <signal>released()</signal>
   <receiver>ClientApplication</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
